supabase/migrations/10_spec_ext.sql

```diff
@@ -85,7 +85,19 @@ comment on view combined_grants_ext is
 -- Extended view of live catalog specifications.
 create view live_specs_ext as
 select
-  l.*,
+  distinct on (l.catalog_name) catalog_name,
```

Comment:
The join with `auth_roles` produces duplicate catalogs, because there are multiple capabilities for each role, e.g. if I am admin, there will be a join that produces capability: read, one capability: write. There is a better way to do this I'm sure, I'll take a look

Reply:
Okay, I moved the aggregation to the join 👍🏽